crates/typst-pdf/src/lib.rs

```diff
@@ -124,6 +124,8 @@ struct PdfContext<'a> {
     pattern_map: Remapper<PdfPattern>,
     /// Deduplicates external graphics states used across the document.
     extg_map: Remapper<ExtGState>,
+    /// Deduplicates emojis
+    emoji_font_map: EmojiFontMap,
```

Comment:
I feel like this field and all the related types should be named differently, as "emoji" is a bit restrictive compared to what it can actually do (even if that is probably the main use case). But I couldn't come up with a better name. Suggestions welcome :grin:

Reply:
Maybe `ColorFontMap`?

Reply:
Okay, I'll go for that, thanks :)